.github/workflows/make_bundle_conda.yml

```diff
@@ -256,7 +256,7 @@ jobs:
       - name: Load signing certificate (MacOS)
         shell: bash -l {0}
         # We only sign pushes to main, nightlies, RCs and final releases
-        if: runner.os == 'macOS' && (github.event_name == 'schedule' || github.event_name == 'push')
+        if: runner.os == 'macOS' # && (github.event_name == 'schedule' || github.event_name == 'push')
```

Comment:
Does that need to be decommented before merge ?

Reply:
Yup! And two more lines like that (notarization and stapling steps).

Reply:
Done :) @Carreau 

If you merge as is, remember to remove the `[skip ci]` part of the commit message so it does run on `main`.